cpp/models/ide_secir/parameters.h

```diff
@@ -90,51 +83,54 @@ struct StateAgeFunction {
      */
     bool operator==(const StateAgeFunction& other) const
     {
-        return (typeid(*this).name() == typeid(other).name() && m_funcparam == other.get_funcparam());
+        return (typeid(*this).name() == typeid(other).name() && m_parameter == other.get_parameter());
     }
 
     /**
      * @brief Here a pure virtual function is defined that depends on the state_age. 
      *
      * The defined function ususally depends on some function parameter.
      * 
-     * @param state_age Time at which the function is evaluated.
+     * @param[in] state_age Time at which the function is evaluated.
      */
     virtual ScalarType eval(ScalarType state_age) = 0;
 
     /**
-     * @brief Get the m_funcparam object
+     * @brief Get the m_parameter object
      * 
-     * Can be used to access the m_funcparam object, which specifies the used function.
+     * Can be used to access the m_parameter object, which specifies the used function.
      * 
      * @return ScalarType 
      */
-    ScalarType get_funcparam() const
+    ScalarType get_parameter() const
     {
-        return m_funcparam;
+        return m_parameter;
     }
 
     /**
-     * @brief Set the m_funcparam object.
+     * @brief Set the m_parameter object.
      * 
-     * Can be used to set the m_funcparam object, which specifies the used function.
+     * Can be used to set the m_parameter object, which specifies the used function.
+     *
+     *@param[in] new_parameter New paramter for StateAgeFunction.
      */
-    void set_funcparam(ScalarType new_funcparam)
+    void set_parameter(ScalarType new_parameter)
     {
-        m_funcparam = new_funcparam;
+        m_parameter = new_parameter;
     }
 
     /**
-     * @brief Computes max_support of Function depending on time step size dt and some tolerance tol. 
+     * @brief Computes maximum support of the function depending on time step size dt and some tolerance tol. 
```

Comment:
```suggestion
     * @brief Computes the maximum support of the function using time step size dt and some tolerance tol. 
```

Even if I proposed something here yesterday, I thought again if the description is suitable. Is the support really "dependent" on the time step anywhere? I suppose not?!

Reply:
In StateAgeFunction in the basic version of get_max_support we evaluate the function at every time point until the function is below the tolerance. Another way would be to evaluate the function at fixed time points that do not depend in the time step size. But this could lead to an overestimation of the max_support (especially for small time steps) which would lead to more computations in the simulation.

Reply:
Yes. What I meant is that the previous text could lead to the misunderstanding that the maximum support depends on the time step size (which is not the case) but it only uses the time step size to determine it. That's why I wanted to change the description. You can also use another describtion than my suggestion.